### ISSUE TITLE
Pretrained model

### ISSUE THREAD

**Issue by Carl-Yang98** [TencentARC/SEED-Story#14]
Besides the pretrained models you mentioned, maybe it's necessary to prepare the Llama-2-7b-hf ? as you mentioned in configs/clm_models/llama2chat7b_lora.yaml line 5:
`pretrained_model_name_or_path: pretrained/Llama-2-7b-hf`

**Comment by Magicalppj** [comment]
Same problem, faild to run the demo

**Comment by Carl-Yang98** [comment]
yep, I use the  inference code and met some problems.  But i almost solve thees problems that I met, and run the inference.
Maybe you should modify some filename and download the pre-trained model 'Llama-2-7b-hf'

**Comment by AndysonYs** [comment]
Hi! Thanks for reminders. I have updated the Llama requirement in the readme file.

**Comment by AndysonYs** [comment]
@Carl-Yang98 @Magicalppj Let me know if you still meet some other problems.

**Comment by Carl-Yang98** [comment]
can you show me some inference results? since I run the inference code and got some unexpected result pic. 
Besides, is there anything else that needs to be modified in the inference code?

**Comment by AndysonYs** [comment]
The generated results are saved in a file tree like this. For every user input, we have a folder to save all the generated results.
![image](https://github.com/user-attachments/assets/98d06248-3c70-44e7-8028-8effe9da531f)

Generated images are the jpg files. Generated texts are presented as subtitle in the jpg images. Just like the format below.
![image](https://github.com/user-attachments/assets/4478ea05-6d99-4d91-8865-d987cbab13c1)

**Comment by AndysonYs** [comment]
You can pack all your results in a zip file and send it to my email if you want. The quality of most of the generated images should be on par with this and the cases presented in my paper.

**Comment by Carl-Yang98** [comment]
Thanks for your reply! 
I run the inference code in cpu and disable the xformers, maybe both of them lead to the unexpected result. I will run it in GPU and check the result.
